website/docs/guides/version_compatibility.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: "appgatesdp"
+page_title: "Version compatibility"
+sidebar_current: "docs-appgate-guide-version-compatibility"
+description: |-
+  Version compatibility
+---
+
+## Version compatibility
+
+The terraform provider tries to maintain full support for all [supported version](https://www.appgate.com/support/software-defined-perimeter-support). Depending on your appgate SDP appliance version, the configuration is diffrent.
+You need to specific the `client_version` if you are not running the latest supported version.
```

Comment:
```suggestion
You need to specify the `client_version` if you are not running the latest supported version.
```

Reply:
Is `client_version` a standard terraform term? Else I would go with something like `api_version` to avoid confusion with actual appgate clients.

Reply:
no, its just a variable name for the peer api version, used in the Accept header,  `application/vnd.appgate.peer-v15+json`, sure we can update this, but that is a bigger refactor that I could do in another PR.